### PR TITLE
[SDK-3215] Added configuring auth0 credentials section

### PR DESCRIPTION
**Pull request by poovamraj**
- Restructured old section to highlight Web.config file
- Added `Configure the Middleware` as a new section
- Rearranged the file order

### CODE REVIEW

File: articles/quickstart/webapp/aspnet-owin/interactive.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Auth0 allows you to quickly add authentication and gain access to user profile i
   returnTo: 'http://localhost:3000'
 }) %>
 
-## Install and Configure the Middleware {{{ data-action=code data-code="Startup.cs#9:13" }}}
+## Setup the Project {{{ data-action=code data-code="Web.config#1:7" }}}
```
Comment (frederikprijck) [inline articles/quickstart/webapp/aspnet-owin/interactive.md]: I believe Setup the Project sounds like we are helping them set up the entire project, while all were are doing is help them install and configure the middleware.

**Review by frederikprijck — CHANGES_REQUESTED** [review]

**Review by frederikprijck — APPROVED** [review]

**Review by avanscoy — APPROVED** [review]
Adding my approval. Needs reviewed by @poovamraj before merge.

**Comment by frederikprijck** [comment]
@avanscoy As the author of the PR, I think @poovamraj can't approve this PR. Are you okay with that I merge this?

**Comment by avanscoy** [comment]
Sure. I wanted to make sure the changes I made didn't hurt the content, but as long as it's fine, go for it.

**Comment by frederikprijck** [comment]
Ooooh that does make sense. Sorry about that, @poovamraj could you let us know if the changes look fine for you regarding that? Feel free to merge if they do!

**Comment by poovamraj** [comment]
Hey @frederikprijck and @avanscoy, I have checked it out and this is good to go. I am merging this.

**Pull request merged**